README.adoc

```diff
@@ -53,13 +53,17 @@ If Antora is installed globally, you should also add this module globally using
 
 ====
 
-When generating your documentation site again, an index file should be created in the output directory `public/search_index.json`.
+When generating your documentation site again, an index file will be created at the root of your output directory,
+which depends on the value of `output.dir` in your playbook.
+For the https://docs.antora.org/antora/2.0/playbook/configure-output/#default-output-dir[default output dir],
+that will be `build/site/search_index.json`.
```

Comment:
@Mogztter The default is `build/site`, so I win!!  But seriously, if you want this change, that's not a problem.

Reply:
That's perfect :+1: